data.braincommons.org/manifest.json

```diff
@@ -62,7 +62,7 @@
     "environment": "bhcprodv2",
     "hostname": "data.braincommons.org",
     "revproxy_arn": "arn:aws:acm:us-east-1:728066667777:certificate/3dfa6ec9-320b-4ce6-ab83-967e286a4d76",
-    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/bhcdictionary/0.6.3/schema.json",
+    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/bhcdictionary/0.8.1/schema.json",
```

Comment:
I think it should match this? https://github.com/uc-cdis/cdis-manifest/blob/master/data-staging.braincommons.org/manifest.json#L57